feature/collection-sheet/src/main/java/com/mifos/feature/individual_collection_sheet/new_individual_collection_sheet/ui/NewIndividualCollectionSheetScreen.kt

```diff
@@ -0,0 +1,283 @@
+                    onOptionSelected = { index, value ->
+                        state.staffList[index].id?.let {
+                            staffId = it
+                        }
+                        selectedStaff = value
+                    },
+                    label = R.string.feature_collection_sheet_staff,
+                    options = state.staffList.map { it.displayName.toString() }
+                )
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    Button(
+                        onClick = {
+                            keyboardController?.hide()
+                            generateCollection(
+                                officeId,
+                                staffId,
+                                SimpleDateFormat("dd MMMM yyyy", Locale.getDefault()).format(
+                                    repaymentDate
+                                )
+                            )
+                        },
+                        modifier = Modifier
+                            .weight(1f)
+                            .padding(16.dp),
+                        contentPadding = PaddingValues(),
+                        enabled = selectedStaff != "" && selectedOffice != "",
+                        colors = ButtonDefaults.buttonColors(
+                            containerColor = if (isSystemInDarkTheme()) BluePrimaryDark else BluePrimary
+                        )
+                    ) {
+                        Text(
+                            text = stringResource(id = R.string.feature_collection_sheet_generate),
+                            fontSize = 16.sp
+                        )
+                    }
+                    Button(
+                        onClick = {
+                            selectedOffice = ""
+                            repaymentDate = System.currentTimeMillis()
+                            selectedStaff = ""
+                        },
+                        modifier = Modifier
+                            .weight(1f)
+                            .padding(16.dp),
+                        contentPadding = PaddingValues(),
+                        colors = ButtonDefaults.buttonColors(
+                            containerColor = if (isSystemInDarkTheme()) BluePrimaryDark else BluePrimary
+                        )
+                    ) {
+                        Text(
+                            text = stringResource(id = R.string.feature_collection_sheet_clear),
+                            fontSize = 16.sp
+                        )
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Preview(showSystemUi = true)
+@Composable
+private fun NewIndividualCollectionSheetPreview() {
+    NewIndividualCollectionSheetScreen(
+        NewIndividualCollectionSheetUiState(),
```

Comment:
In any way this will not generate the proper UI state wise preview, connect with @AvneetSingh2001 or @PratyushSingh07 fix this.

Reply:
ok

Reply:
sure bhaiya